psi4/src/export_mints.cc

```diff
@@ -1542,7 +1542,10 @@ void export_mints(py::module& m) {
                                                         "Extracts information from a wavefunction object, \
                                                                           and writes it to an FCHK file")
         .def(py::init<std::shared_ptr<Wavefunction>>())
-        .def("write", &FCHKWriter::write, "Write wavefunction information to file", "filename"_a);
+        .def("write", &FCHKWriter::write, "Write wavefunction information to file", "filename"_a)
+        .def("SCF_Dtot",&FCHKWriter::SCF_Dtot,py::return_value_policy::reference_internal)
+        .def("recompute_scf_density", &FCHKWriter::recompute_scf_density, "Re-compute the SCF density from wfn.Ca/Cb if True, or use SCF density from reference_wavefunction if False")
+        .def("set_postscf_density_label", &FCHKWriter::set_pHF_density_label, "Set base label for post-SCF density, e.g. ' CC Density'.", "label"_a);
```

Comment:
Why do two out of four string constants have a `_a` suffix? What does it even do?

Reply:
It's a shorthand to inform about the name of the argument:
https://pybind11.readthedocs.io/en/stable/basics.html#keyword-arguments

And for `recompute_scf_density` I missed to add an argument name.

Reply:
added missing argument name